src/python/pants/releases/packages.py

```diff
@@ -89,14 +91,19 @@ def find_platform_name():
   return get_platform().replace("-", "_").replace(".", "_")
 
 
-core_packages = {
-  Package(
-    "pantsbuild.pants",
-    "//src/python/pants:pants-packaged",
-    bdist_wheel_flags=("--python-tag", "cp27", "--plat-name", find_platform_name()),
-  ),
-  Package("pantsbuild.pants.testinfra", "//tests/python/pants_test:test_infra"),
-}
+def core_packages(py3):
+  # N.B. When releasing with Python 3, we constrain the ABI (Application Binary Interface) to cp36 to allow
```

Comment:
love the comment details man

Reply:
Fun fact, you all taught me what NB means! And taught me that while self-documenting code is great, with highly technical situations like this, comments with links to standards and upstream issues are a great thing.